modules/docs/lib/stackblitzFiles/appFile.ts

```diff
@@ -0,0 +1,35 @@
+export const appFile = `import * as React from 'react';
```

Comment:
What if you changed these to real files and imported them as `!raw` instead? It would be easier to read and verify any mistakes.

Reply:
done